config.json

```diff
@@ -1124,6 +1124,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 7
+      },
+      {
+        "slug": "spiral-matrix",
+        "name": "Spiral Matrix",
+        "uuid": "d5569902-5815-4b05-ab13-13826fef7826",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5
```

Comment:
Do you think this difficulty is at a proper level? I have not done this exercise, but your code is not a 5 compared to the rest. Is there a simpler way, that just looks less idiomatic, or is it one of the more complicated exercises in your opinion?